notifier/notifier.go

```diff
@@ -610,7 +610,10 @@ type alertmanager interface {
 	url() *url.URL
 }
 
-type alertmanagerLabels struct{ labels.Labels }
+type alertmanagerLabels struct {
+	labels.Labels
```

Comment:
can we have an explicit field name here?

Reply:
OK